lxd/db/cluster/identities.go

```diff
@@ -290,15 +292,17 @@ func (i Identity) Subject() (string, error) {
 }
 
 // ToAPI converts an Identity to an api.Identity, executing database queries as necessary.
-func (i *Identity) ToAPI(ctx context.Context, tx *sql.Tx) (*api.Identity, error) {
+func (i *Identity) ToAPI(ctx context.Context, tx *sql.Tx, canViewGroup auth.PermissionChecker) (*api.Identity, error) {
```

Comment:
@markylaing the note for the forthcoming openga server, but can auth.PermissionChecker remain returning only a bool when each call to it could fail with an error (e.g. when consulting the DB)?

Reply:
The permission checker will still only return a boolean. Getting the permission checker can fail, but we're not getting one here, just passing one in.

Reply:
ok cool - so this remains true even with openfga?